units/src/amount/mod.rs

```diff
@@ -202,10 +202,12 @@ const INPUT_STRING_LEN_LIMIT: usize = 50;
 
 /// Parses a decimal string in the given denomination into a satoshi value and a
 /// [`bool`] indicator for a negative amount.
+///
+/// The `bool` is only needed to distinguish -0 from 0.
 fn parse_signed_to_satoshi(
     mut s: &str,
     denom: Denomination,
-) -> Result<(bool, u64), InnerParseError> {
```

Comment:
It's strange to see `u64` replaced by `SignedAmount` in the diff.  I'd have expected to see `i64` replaced by `SignedAmount`.

Reply:
It's explained in the commit message. I suggest you read them.

Reply:
I read through them yesterday although I guess this one is new today.  Today I was paging through the diff to see what change although I should pay closer attention to what commit messages changed as well.

Reply:
Discussions also often provide insight.